lib/matplotlib/path.py

```diff
@@ -528,22 +587,39 @@ def contains_path(self, path, transform=None):
             transform = transform.frozen()
         return _path.path_in_path(self, None, path, transform)
 
-    def get_extents(self, transform=None):
-        """
-        Return the extents (*xmin*, *ymin*, *xmax*, *ymax*) of the path.
+    def get_extents(self, transform=None, **kwargs):
+        """Get size of Bbox of curve (instead of Bbox of control points).
+
+        Parameters
+        ----------
+        transform : matplotlib.transforms.Transform, optional
+            Transform to apply to path before computing extents, if any.
+        kwargs : Dict[str, object]
+            Forwarded to self.iter_bezier.
 
-        Unlike computing the extents on the *vertices* alone, this
-        algorithm will take into account the curves and deal with
-        control points appropriately.
+        Returns
+        -------
+        bbox : matplotlib.transforms.Bbox
+            The extents of the path Bbox([[xmin, ymin], [xmax, ymax]])
         """
         from .transforms import Bbox
-        path = self
         if transform is not None:
-            transform = transform.frozen()
-            if not transform.is_affine:
-                path = self.transformed(transform)
-                transform = None
-        return Bbox(_path.get_path_extents(path, transform))
+            self = transform.transform_path(self)
+        maxi = 2  # [xmin, ymin, *xmax, ymax]
```

Comment:
Unused?

Reply:
don't know why flake8 didn't catch this...